draft-irtf-cfrg-ristretto255-decaf448.md

```diff
@@ -485,13 +468,11 @@ The one-way map on an input string b proceeds as follows:
 2. Compute P2 as `MAP(b[32:64])`.
 3. Return P1 + P2.
 
-The MAP function is defined on a bytestring of 32 bytes in length as:
+The MAP function is defined on a 32-byte string as:
 
-1. Interpret the input bytestring as in integer r in little-endian
-   convention, then reduce it modulo 2^255 (this is equivalent to
-   "masking out the most significant bit" as in [@RFC7748],
-   section 5). The resulting integer, which is lower than 2^255,
-   is then further reduced modulo p, yielding a field element t.
+1. First, interpret the string as an integer r in little-endian
+   representation. Reduce r modulo 2^255, then further reduce it
+   modulo p, to obtain a field element t.
    * Note: similarly to [@RFC7748] field element decoding, the most
      significant bit of the representation of r is masked.
```

Comment:
Here, do we mean to leave open the option of doing some kind of reduction mod 2^255 other than masking the MSB? The previous language made it seem that the intended operation is masking, but this revision leaves you to infer it.

Reply:
Addressed in d11463d6a74ebe9c8d752fb5bc37a210aaf97d32.